pyccel/codegen/pipeline.py

```diff
@@ -55,7 +55,9 @@ def execute_pyccel(fname, *,
                    debug         = False,
                    accelerators  = (),
                    output_name   = None,
-                   compiler_export_file = None):
+                   compiler_export_file = None,
+                   is_conda_warnings_disabled = False,
+                   is_conda_warnings_detailed = False):
```

Comment:
Please update the docstring

Reply:
https://github.com/pyccel/pyccel/blob/e1f3ebeea7a4b621de7f03408aa23eda26a87764/pyccel/codegen/pipeline.py#L61-L113